airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -33,9 +43,12 @@
     UpdatedCursorIncrementalStripeStream,
 )
 
-_MAX_CONCURRENCY = 3
+logger = logging.getLogger("airbyte")
+
+_MAX_CONCURRENCY = 20
```

Comment:
[the limit in the spec ](https://github.com/airbytehq/airbyte/pull/32284/files#diff-31ac2b459174644b10c8b738b4fe98173c4cdfded04d9db4f65a4cfa67d0ab00R64)should be aligned with this limit